tests/fixtures/noqa/noqa.py

```diff
@@ -692,3 +692,14 @@ def consecutive_yields():
     'wrong',
 ]
 *numbers, = [4, 7]  # noqa: WPS356
+
+
+class SomeOtherClass(object):
+    def many_raises_instance_method(self, parameter):  # noqa: WPS235
```

Comment:
Can we use a function and not a method? We try to bring the minimal example here.